Makefile

```diff
@@ -31,6 +31,7 @@ mod: ## go mod tidy
 inst: ## go install tools
 	$(call print-target)
 	cd tools && go install $(shell cd tools && go list -e -f '{{ join .Imports " " }}' -tags=tools)
+	pip install --user yamllint
```

Comment:
Not everyone has python and wants to install yamllint globally.

How about installing it via docker in a virtual-env? 

Here is an example how we install and use a Python tool in OpenTelemetry Go: https://github.com/open-telemetry/opentelemetry-go/blob/f88533381ba23b17bb33f22a762053fb35fd80e7/Makefile#L241-L243